src/YesSql.Core/Commands/BatchCommand.cs

```diff
@@ -9,8 +9,6 @@ namespace YesSql.Commands
 {
     public class BatchCommand : IIndexCommand
     {
-        public static int DefaultBuilderCapacity = 10 * 1024;
```

Comment:
This is a static attribute that allows developers to modify it, supposedly to adapt the maximum number of parameters for different databases

Reply:
The issue is that this property is never used in any concrete way. It's possible that at some point I actually used a StringBuilder, but this is not the case anymore and the current implementation (string.Concat) is actually better than StringBuilder. So this should be removed. This is a breaking change though and maybe it should be made obsolete instead.

Reply:
Obsoleted it instead.